README.md

```diff
@@ -25,6 +25,16 @@ _Optional_ The merge method you would like to use (squash, merge, rebase). Defau
 
 _Optional_ An arbitrary message that you'd like to comment on the PR after it gets auto-merged. This is only useful when you're recieving too much of noise in email and would like to filter mails for PRs that got automatically merged.
 
+### `enable-auto-merge`
+
+_Optional_ If `true`, the PR is marked as auto-merge and will be merged by github when status checks are satisfied. Default to `false`.
+
+_NOTE_ This feature only works when all of the following conditions are met.
+
+- The repository enables auto-merge.
+- The pull request base must have a branch protection rule.
+- The pull request's status checks are not yet satisfied.
```

Comment:
It is not totally clear to me the workflow here. Let me try to recap the use case:

- dependabot opens a PR
- the CI run and it is green
- the automerge step is executed and the PR is marked as auto-merge-by-gh
- the PR is not merged because it is green (?)

Another use case:

- dependabot opens a PR
- the CI run and it is red
- the automerge step is executed and the PR is marked as auto-merge-by-gh
- the developer must fix the PR due breaking changes, by using the same branch as dependabot
- the CI is green and GH auto merge the PR

Reply:
In the first use case, step 3 fails because github won't allow us to mark green PRs as auto-merge enabled. The second should work though I didn't try that.